### PR TITLE
8710 feature: Integrate design system foundations

### PR DESCRIPTION
**Pull request by creido-welly**
Relates to:
* wellcometrust/corporate#8710 
* wellcometrust/corporate#9030

## Context
Initial work to integrate design system in Wellcome.org

* Latest DS package installed
* Core styles imported
* Colour rgba values replaced with DS variables

## Additional updates
* Additional unit tests to verify that rich text output neutralises potential XSS attacks

### CODE REVIEW

**Review by chriddell — APPROVED** [review]

**Comment by creido-welly** [comment]
@chriddell tnx for reviewing this and related PRs

**Comment by creido-welly** [comment]
**DO NOT MERGE**

Before this branch can be merged and a new package published, the consuming repo, [corporate-react](wellcometrust/corporate-react), needs updating to use the design system colour variables.

See wellcometrust/corporate#9036 and wellcometrust/corporate#9037 for details.

**Pull request merged**
